lib/nodeMap.js

```diff
@@ -190,6 +189,13 @@ api.createNodeMap = (input, graphs, graph, issuer, name, list) => {
           subject, property, {'@id': id},
           {propertyIsArray: true, allowDuplicate: false});
         api.createNodeMap(o, graphs, graph, issuer, id);
+      } else if(graphTypes.isValue(o)) {
+        //if('@type' in o) {
+        //  o['@type'] = o['@type'][0];
+        //}
```

Comment:
I don't recall what this commented code was for. Anyone know if it is needed?

Reply:
It seemed to be for a case that can't happen, as it's handled later in the algorithm, so this would never happen.